packages/styled-docs/pages/spinner.mdx

```diff
@@ -0,0 +1,62 @@
+# Spinner
+
+A spinner can be used to display the loading state when the part of the page is waiting for the results.
+
+## Import
+
+```js
+import Spinner from '@trendmicro/react-styled-core/Spinner';
+// or
+import { Spinner } from '@trendmicro/react-styled-core';
+```
+
+## Usage
+
+```jsx
+<Spinner />
+```
+
+### Colors
+
+```jsx
+<Stack direction="row" spacing="6x">
+  <Spinner color="red:50" />
+  <Spinner color="red" />
+  <Spinner color="white" />
+  <Spinner color="rgba(255, 167, 196, 1)" />
+  <Spinner color="#8f41e9" />
+</Stack>
+```
+
+### Sizes
```

Comment:
Use the `size` prop to change the size of the `Spinner`. You can set the value to `xs`, `sm`, `md`, `lg`, or `xs`. The corresponding stroke width is listed as below:
| Size | Stroke Width |
| :---- | :------------- |
| xs | 2px |
| sm | 2px |
| md | 4px |
| lg | 4px |
| xl | 4px |